src/components.rs

```diff
@@ -23,7 +23,7 @@ pub use venue::*;
 #[derive(Debug, Default, PartialEq, Eq, Clone)]
 pub(crate) struct InnerComponent {
     pub properties: BTreeMap<String, Property>,
-    pub multi_properties: Vec<Property>,
+    pub multi_properties: BTreeMap<String, Vec<Property>>,
```

Comment:
Didn't you want to go with a multimap here?

Reply:
We still can, it does add another dependencies and I thought this approach might be more simple. It also keeps `multi_properties` more similar to `properties` in type. Happy to change it to multimap if you prefer.